vars/syncRepo.groovy

```diff
@@ -1,19 +1,23 @@
-def call(String REPO_NAME) {
+def call(String REPO_NAME, String PROBUILD) {
     node('jenkins') {
         withCredentials([sshUserPrivateKey(credentialsId: 'repo.ci.percona.com', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
             sh """
                 ssh -o StrictHostKeyChecking=no -i ${KEY_PATH} ${USER}@repo.ci.percona.com ' \
                     set -o errexit
                     set -o xtrace
-
-                    # Update /srv/repo-copy/version
-                    date +%s > /srv/repo-copy/version
+                    if [ ${PROBUILD} = YES ]; then
+                        PRO_FOLDER="private/"
+                    else
+                        PRO_FOLDER=""
+                    fi
+                    # Update /srv/repo-copy/${PRO_FOLDER}/version
+                    date +%s > /srv/repo-copy/\${PRO_FOLDER}/version
```

Comment:
IMHO the forward slash should be removed here. It's already provided in PRO_FOLDER variable.

Reply:
@adivinho Could you please check now?